spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
```

Comment:
You are right. I had to create that myself. It is just me being a little rusty. Sorry. Keep it